### DISCUSSION TITLE
Error: The request specifies an invalid filter parameter

### DISCUSSION THREAD

**Discussion by Marx2** [mxpv/podsync#733] [General]
On some of streams I'm getting an error:
`time="2022-02-07T10:21:28+01:00" level=error msg="failed to update feed: https://www.youtube.com/channel/UCcPuUNlUF4DWQ5e0hrE4cug" error="update failed: failed to query video descriptions: googleapi: Error 400: The request specifies an invalid filter parameter., invalidFilters"
`
From googling it seems that sometimes podsync is calling youtube-dl with wrong parameter. It happens only on some channels. Those channels were working ok in the past.
Any idea how to fix it?

**Comment by Rumik** [comment]
I'm also having this problem.

**Comment by bcarney66** [comment]
I corrected this by dropping page_size to any number under 50.

`page_size = 48`

**Comment by Elycoo** [comment]
If I want to maintain it above 50?
It should be possible with multiple requests. See [here.](https://stackoverflow.com/questions/60412473/using-the-youtube-api-im-getting-invalid-filter-parameter-error-what-is-wron)

**Comment by Elycoo** [comment]
Sorry, just now notice that I have an old version of Podsync and this has been discussed already [here](https://github.com/mxpv/podsync/issues/306).
Thanks!
